contrib/mesos/pkg/minion/tasks/task.go

```diff
@@ -29,7 +29,10 @@ import (
 	"k8s.io/kubernetes/contrib/mesos/pkg/runtime"
 )
 
-const defaultTaskRestartDelay = 5 * time.Second
+const (
+	defaultTaskRestartDelay = 5 * time.Second
+	defaultKillGracePeriod  = 5 * time.Second
```

Comment:
agreed. we can pick a some higher (arbitrary) value but without knowing when the slave will forcibly kill us it seems a somewhat futile exercise.